morph_tool/spatial.py

```diff
@@ -22,9 +22,9 @@ def point_to_section_segment(neuron, point):
 
```

Comment:
Since we use `np.allclose` I am wondering if we should add `rtol` and `atol` parameters to the function signature? Maybe someone will want to play with these values someday?

Reply:
done